<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MassMain</class>
 <widget class="QMainWindow" name="MassMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>823</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>823</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1024</width>
    <height>823</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MassMain</string>
  </property>
  <widget class="QWidget" name="MASS">
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>768</height>
    </size>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1001</width>
      <height>771</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="startthem">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="stopthem">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLineEdit" name="corobie">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="ustawienia">
         <property name="text">
          <string>Ustawienia</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="Usta" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>999</width>
          <height>33</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="usta_bar">
         <item>
          <widget class="QLabel" name="przedrostek_label">
           <property name="text">
            <string>Przedrostek</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="przedrostek">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string>out_</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="jakosc_label">
           <property name="text">
            <string>Jakość:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="jakosc_bar">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Szerokosc_label">
           <property name="text">
            <string>Szerokość</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Szerokosc">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="inputMask">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Wysokosc_label">
           <property name="text">
            <string>Wysokość</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="Wysokosc">
           <property name="maximumSize">
            <size>
             <width>60</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPliki">
    <property name="title">
     <string>Pliki</string>
    </property>
    <addaction name="actionWczytaj_Plik"/>
    <addaction name="actionWczytaj_Katalog"/>
    <addaction name="separator"/>
    <addaction name="actionResetuj"/>
    <addaction name="actionCzysc"/>
    <addaction name="separator"/>
    <addaction name="actionZamknij"/>
   </widget>
   <widget class="QMenu" name="menuFormat">
    <property name="tearOffEnabled">
     <bool>false</bool>
    </property>
    <property name="title">
     <string>Format</string>
    </property>
    <addaction name="actionBMP"/>
    <addaction name="actionJPG"/>
    <addaction name="actionJPEG"/>
    <addaction name="actionPNG"/>
    <addaction name="actionJP2"/>
    <addaction name="separator"/>
    <addaction name="actionICO"/>
    <addaction name="actionPIC"/>
    <addaction name="separator"/>
    <addaction name="actionRGB"/>
    <addaction name="actionRGBA"/>
    <addaction name="separator"/>
    <addaction name="actionTif"/>
    <addaction name="actionTIIFF"/>
    <addaction name="separator"/>
    <addaction name="actionTGA"/>
    <addaction name="actionPCX"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
    <addaction name="actionO_qt"/>
    <addaction name="separator"/>
    <addaction name="actionO_programie"/>
   </widget>
   <addaction name="menuPliki"/>
   <addaction name="menuFormat"/>
   <addaction name="menuInfo"/>
  </widget>
  <action name="actionWczytaj_Katalog">
   <property name="text">
    <string>Wczytaj Katalog</string>
   </property>
  </action>
  <action name="actionZamknij">
   <property name="text">
    <string>Zamknij</string>
   </property>
  </action>
  <action name="actionJPEG">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>JPEG</string>
   </property>
   <property name="autoRepeat">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPNG">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>PNG</string>
   </property>
  </action>
  <action name="actionBMP">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>BMP</string>
   </property>
  </action>
  <action name="actionO_qt">
   <property name="text">
    <string>O Qt ...</string>
   </property>
  </action>
  <action name="actionO_programie">
   <property name="text">
    <string>O programie...</string>
   </property>
  </action>
  <action name="actionAutor">
   <property name="text">
    <string>Autor</string>
   </property>
  </action>
  <action name="actionCzysc">
   <property name="text">
    <string>Czysc</string>
   </property>
  </action>
  <action name="actionICO">
   <property name="text">
    <string>ICO</string>
   </property>
  </action>
  <action name="actionWczytaj_Plik">
   <property name="text">
    <string>Wczytaj Plik</string>
   </property>
  </action>
  <action name="actionJPG">
   <property name="text">
    <string>JPG</string>
   </property>
  </action>
  <action name="actionBW">
   <property name="text">
    <string>BW</string>
   </property>
  </action>
  <action name="actionEPS">
   <property name="text">
    <string>EPS</string>
   </property>
  </action>
  <action name="actionEPSF">
   <property name="text">
    <string>EPSF</string>
   </property>
  </action>
  <action name="actionRGB">
   <property name="text">
    <string>RGB</string>
   </property>
  </action>
  <action name="actionRGBA">
   <property name="text">
    <string>RGBA</string>
   </property>
  </action>
  <action name="actionTGA">
   <property name="text">
    <string>TGA</string>
   </property>
  </action>
  <action name="actionPIC">
   <property name="text">
    <string>PIC</string>
   </property>
  </action>
  <action name="actionJP2">
   <property name="text">
    <string>JP2</string>
   </property>
  </action>
  <action name="actionPCX">
   <property name="text">
    <string>PCX</string>
   </property>
  </action>
  <action name="actionPPM">
   <property name="text">
    <string>PPM</string>
   </property>
  </action>
  <action name="actionTif">
   <property name="text">
    <string>TIF</string>
   </property>
  </action>
  <action name="actionTIIFF">
   <property name="text">
    <string>TIFF</string>
   </property>
  </action>
  <action name="actionEPSI">
   <property name="text">
    <string>EPSI</string>
   </property>
  </action>
  <action name="actionPrzedrostek">
   <property name="text">
    <string>Przedrostek</string>
   </property>
  </action>
  <action name="actionResetuj">
   <property name="text">
    <string>Resetuj</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionBMP</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionJPEG</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPNG</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionCzysc</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>czyscliste()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startthem</sender>
   <signal>clicked()</signal>
   <receiver>MassMain</receiver>
   <slot>go()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>69</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionICO</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>417</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWczytaj_Plik</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>load_file()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionWczytaj_Katalog</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>load_dir()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZamknij</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionO_qt</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>oqt()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionJPG</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionO_programie</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>oprogramie()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionJP2</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPIC</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTIIFF</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTif</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRGB</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionRGBA</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionTGA</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionPCX</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>ZmienFormat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopthem</sender>
   <signal>clicked()</signal>
   <receiver>MassMain</receiver>
   <slot>stopthem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>145</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ustawienia</sender>
   <signal>clicked()</signal>
   <receiver>MassMain</receiver>
   <slot>ustawienia()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>964</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>jakosc_bar</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>MassMain</receiver>
   <slot>zmienjakosc(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>412</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionResetuj</sender>
   <signal>activated()</signal>
   <receiver>MassMain</receiver>
   <slot>reset()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listWidget</sender>
   <signal>itemDoubleClicked(QListWidgetItem*)</signal>
   <receiver>MassMain</receiver>
   <slot>one(QListWidgetItem*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>411</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ZmienFormat()</slot>
  <slot>czyscliste()</slot>
  <slot>go()</slot>
  <slot>load_file()</slot>
  <slot>load_dir()</slot>
  <slot>oqt()</slot>
  <slot>oprogramie()</slot>
  <slot>oautorze()</slot>
  <slot>stopthem()</slot>
  <slot>pausethem()</slot>
  <slot>one(QListWidgetItem*)</slot>
  <slot>ustawienia()</slot>
  <slot>zmienjakosc(int)</slot>
  <slot>reset()</slot>
 </slots>
</ui>
